app/src/main/java/mozilla/lockbox/store/AccountStore.kt

```diff
@@ -235,8 +237,22 @@ open class AccountStore(
 
         webView.clearCache(true)
         clearLogs()
+        removeDeviceFromFxA()
```

Comment:
Why did we lose the `DataStoreAction.Reset`?

Reply:
It was redundant here (per our discussion when I added the `clearLogs()`) but I forgot to remove it in that PR before it was merged.